src/components/docs-sidebar-nav-item.js

```diff
@@ -170,7 +180,7 @@ class DocsSidebarNavItem extends React.Component {
             <div className={`${collapseClassesBase}content`}>
               <ul className="DocsSidebar--nav-subnav" depth={depth} style={{'--depth': depth}}>
                 {node.children.map(node => (
-                  <DocsSidebarNavItem
+                  <DocsSidebarNavItemWrapper
```

Comment:
When possible, we always prefer to export a component with the same name as the file (less casing). So instead of wrapping `<DocsSidebarNavItem/>` in `<DocsSidebarNavItemWrapper/>`, maybe rename `<DocsSidebarNavItem/>` to `<DocsSidebarNavItemContent/>` and then name the wrapper just `<DocsSidebarNavItem/>`.

Reply:
got it